src/main/java/io/github/thebusybiscuit/slimefun4/implementation/items/electric/gadgets/Multimeter.java

```diff
@@ -54,6 +54,7 @@ public ItemUseHandler getItemHandler() {
                     p.sendMessage("");
                     Slimefun.getLocalization().sendMessage(p, "messages.multimeter", false, str -> str.replace("%stored%", stored).replace("%capacity%", capacity));
                     p.sendMessage("");
+                    component.sendMultimeterInfo(p, l);
```

Comment:
Maybe extract this into the default method:
`Slimefun.getLocalization().sendMessage(p, "messages.multimeter", false, str -> str.replace("%stored%", stored).replace("%capacity%", capacity));`

```suggestion
                    component.sendMultimeterInfo(p, l);
                    p.sendMessage("");
```

This way the padding is kept consistently and it allows you to specify whether your extra info should be added above or below the default text.